CHANGELOG.md

```diff
@@ -34,6 +43,8 @@ Generated on 2020-11-30
 ### Performance
 |||
 |:---|:---|
+|[#1048](https://github.com/NVIDIA/spark-rapids/issues/1048)|[FEA] Benchmark runner should upload summary file and Spark event log to s3|
```

Comment:
IMO the performance section of the release notes should highlight changes that directly impact the performance of the product.  Most of the things listed here are not that.

Reply:
@sameerz plz help check the labels here, thx!

Reply:
I asked @pxLi to modify the changelog script to not pick up the `benchmark` label for performance features.